src/main/java/org/hydev/mcpm/client/uninstallUseCase/UninstallInput.java

```diff
@@ -0,0 +1,9 @@
+package org.hydev.mcpm.client.uninstallUseCase;
+
```

Comment:
Seeing as this is unfinished, an implementation of the input as a record should work.

Reply:
We can use the getSearchReult method in the database manager in the installer folder to help you check if the plugin exists or not.